misago/users/tests/test_anonymizeoldips.py

```diff
@@ -0,0 +1,43 @@
+import datetime
+import pytz
+
+from django.contrib.auth import get_user_model
+from django.core.management import call_command
+from django.test import TestCase
+from django.utils.six import StringIO
+from django.utils import timezone
+
+from misago.conf import settings
+from misago.users.management.commands import anonymizeoldips
+from misago.core.utils import ANONYMOUS_IP
```

Comment:
Those imports are not ordered alphabetically. `misago.users.management.commands` should come last.